tests/status/worktree.c

```diff
@@ -1315,3 +1315,47 @@ void test_status_worktree__at_head_parent(void)
 	git_tree_free(parent_tree);
 	git_status_list_free(statuslist);
 }
+
+void test_status_worktree__override_nested_wildcard_unignore(void)
```

Comment:
You could've saved yourselve quite some pain by using "tests/ignore/status.c" or "tests/ignore/path.c", but somehow nobody discovers that test suite :P It's fine if you don't want to go the extra mile and convert it.

Reply:
Sorry, I saw these tests and the utility methods available. I will move them 😉

Reply:
@pks-t I moved the test to `tests/ignore/status.c`. However, I could not use utilities like `assert_is_ignored` because it does not go through `filesystem_iterator`. I think this is the reason why I left the tests in `worktree.c` initially.

Reply:
Fair enough!